OloEngine/src/OloEngine/Networking/Replication/SnapshotBuffer.cpp

```
@@ -0,0 +1,130 @@
+            {
+                after = &entry;
+            }
+
+            if (before && after)
+            {
+                break;
+            }
+        }
+
+        // If tick is beyond the newest entry, use the two newest
+        if (!after && before)
```

Clang (14.0.6)

[warning] implicit conversion 'const OloEngine::SnapshotBuffer::Entry *' -> bool

(readability-implicit-bool-conversion)